src/legacy/state/orders/updaters/GpOrdersUpdater.ts

```diff
@@ -202,6 +202,8 @@ function _filterOrders(
  * - Persist the new tokens and orders on redux
  */
 export function GpOrdersUpdater(): null {
+  useCleanOrdersStorageOnUnmount()
```

Comment:
One question that I have is.
This will clean up when the app is closed, so we are sure it'll leave in a good state.

What about when loading the app? This doesn't fix the problem in case someone already has the app stuck and cannot open it, no? Or can this fix it?